jdbcSource/src/main/java/io/vantiq/extsrc/jdbcSource/JDBC.java

```diff
@@ -123,15 +123,27 @@ public int processPublish(String sqlQuery) throws VantiqSQLException {
                                 break;
                             case java.sql.Types.DATE:
                                 Date rowDate = queryResults.getDate(i);
-                                row.put(md.getColumnName(i), dfDate.format(rowDate));
+                                if (rowDate != null) {
+                                    row.put(md.getColumnName(i), dfDate.format(rowDate));
+                                } else {
+                                    row.put(md.getColumnName(i), null);
+                                }
```

Comment:
Is there a reason we need the column there?  Since the row's a map, can't we just omit it?